packages/medusa/src/types/routing.ts

```diff
@@ -59,15 +59,17 @@ export interface MedusaRequest<Body = unknown>
   pricingContext?: MedusaPricingContext
 }
 
+export interface AuthContext {
+  actor_id: string
+  actor_type: "api-key" | "user" | "customer" | "unknown"
```

Comment:
thought: this is likely something people will want to extend. Can we come up with something that is more open?

Reply:
@srindom since this is just typings I think we can keep it as-is until we get everything in order internally, and then we can always switch to eg. `string` when it's needed. This is also why I didn't use an enum for now.

Let me add a TODO comment, so we can review it once our auth setup is cleaned up.